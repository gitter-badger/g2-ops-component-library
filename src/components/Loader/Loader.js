import React from 'react'
import { Spinner, SpinnerSize } from 'office-ui-fabric-react/lib/Spinner'
import './Loader.scss'

const Loader = (props) => <Spinner className="spinner" {...props} />

export default Loader
