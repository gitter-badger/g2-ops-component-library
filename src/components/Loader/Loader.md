Copart Loader Example:

```js
<Loader />
```
