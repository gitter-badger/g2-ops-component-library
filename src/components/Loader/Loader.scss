.CopartSpinner {
  .ms-Spinner-circle {
    width: 50px!important;
    height: 50px!important;
    border: 2.5px solid lighten(#1D5AB9, 30%);
    border-top-color: #1D5AB9;
  }
}
