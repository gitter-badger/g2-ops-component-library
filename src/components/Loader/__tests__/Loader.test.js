import React from 'react'
import renderer from 'react-test-renderer'

import Loader from '../Loader'

describe('<Loader />', () => {
  test('should render properly', () => {
    const tree = renderer.create(<Loader />)
    expect(tree).toMatchSnapshot()
  })
})
