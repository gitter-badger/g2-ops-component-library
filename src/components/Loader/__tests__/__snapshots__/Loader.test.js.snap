// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render properly 1`] = `
<div
  className="ms-Spinner root spinner"
>
  <div
    className="ms-Spinner-circle circle ms-Spinner--medium circleIsMedium"
  />
</div>
`;
